examples/WebAPI.Net6/Readme.md

```diff
@@ -0,0 +1,196 @@
+# Application Insights for Kubernetes on WebAPI
+
+This page is a walk-through to enable `Application Insights for Kubernetes` in WebAPI projects, including traditional WebAPI (controller-based) and minimal WebAPIs. The example code is separated in [WebAPI](./WebAPI/) and [MinimalAPI](./MinimalAPI/) respectively.
+
+_To learn more about minimal API, please refer to the [Minimal APIs overview](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-6.0)._
+
+## Prerequisite
+
+* .NET 6 SDK - <https://dot.net>
+    * Verify dotnet version. For example:
+        ```shell
+        dotnet --version
+        6.0.302
+        ```
+    _You might have a slightly different patch version and that should be fine._
+
+* A Kubernetes Cluster that you can manage with `kubectl`.
+    * If you don't have any, there are several options:
+        * [Azure AKS](https://docs.microsoft.com/en-us/azure/aks/)
+        * [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+        * [MiniKube](https://minikube.sigs.k8s.io/docs/start/)
+        * ...
```

Comment:
Don't know if you meant leave this "..." here

Reply:
Thanks for the question. Let me double check.

Reply:
There is actually more than just what I listed. I want to express that it is not an exhausted list. What would you put there?

Reply:
Cool. I'll leave it as is based on our conversation below. Hope that won't be confusing. And I guess we could update it easily.

Reply:
That looks good. Definitely a fair use of the ellipses.